docs/use_forms.md

```diff
@@ -0,0 +1,5 @@
+# Data capture forms
```

Comment:
Leave it out and create a story to develop the missing parts.

Reply:
I rather keep it, I will add a short summary on where the forms live.